compiler/rustc_query_system/src/ich/hcx.rs

```diff
@@ -161,10 +155,7 @@ impl<'a> StableHashingContext<'a> {
 
     #[inline]
     pub fn is_ignored_attr(&self, name: Symbol) -> bool {
-        thread_local! {
-            static IGNORED_ATTRIBUTES: FxHashSet<Symbol> = compute_ignored_attr_names();
-        }
-        IGNORED_ATTRIBUTES.with(|attrs| attrs.contains(&name))
+        ich::IGNORED_ATTRIBUTES.contains(&name)
```

Comment:
Previously this would do a HashSet lookup in `O(1)`. Now it does a `Vec` search in `O(n)`.